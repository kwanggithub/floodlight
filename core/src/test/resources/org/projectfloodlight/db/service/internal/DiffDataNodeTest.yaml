treespaces:
  - name: DiffDataNodeTest
    data_sources:
        - name: config
          implementation_class: org.projectfloodlight.db.data.memory.MemoryDataSource
    data_source_mappings:
      - data_source: config
        predicate: Config
    module_search_paths:
      - path: src/test/resources/org/projectfloodlight/db/service/internal
        recursive: true
    modules:
      - name: DiffDataNodeTest
