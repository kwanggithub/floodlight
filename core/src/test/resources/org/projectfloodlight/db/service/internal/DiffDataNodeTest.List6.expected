{
  "bar|20" : {
    "child1" : "bar",
    "child2" : 20,
    "child3" : "abc",
    "child4" : {
      "child4.1" : 25,
      "child4.2" : "boo"
    }
  },
  "foo|1" : {
    "child3" : "test1"
  }
}